.github/workflows/test.yml

```
@@ -2,7 +2,7 @@ name: test
 
 on:
   push: # Run on pushes to the default branch
-    branches: [main]
+    branches: [main, issue-1710-enable-af3-perf-dag]
```

```suggestion
    branches: [main]
```
Remove before merging